app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -519,6 +520,19 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testOnboardingFragment_nextArrowIcon_hasCorrectContentDescription() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_fragment_next_image_view)).check(
+        matches(
+          withContentDescription(
```

Comment:
as we are setting content description in the XML file and the files are different for each orientation, so I suggest here to add one more test case for the landscape mode. 

```kotlin
testOnboardingFragment_configChange_nextArrowIcon_hasCorrectContentDescription
```

Reply:
Got it...Will do it.